rfc/0076-subdoc-replica-reads.md

```diff
@@ -0,0 +1,183 @@
+# Meta
+
+| Field          | Value                      |
+|----------------|----------------------------|
+| RFC Name       | Sub-Document Replica Reads |
+| RFC ID         | 76                         |
+| Start Date     | 2023-06-20                 |
+| Owner          | Charles Dixon              |
+| Current Status | DRAFT                      |
+| Revision       | #1                         |
+
+# Summary
+[MB-23162](https://issues.couchbase.com/browse/MB-23162) adds support for performing subdocument read operations on replica servers.
+Replica read operations are performed against replica servers with a new ReplicaRead protocol flag.
+it also includes a new, informational, HELLO flag to be negotiated.
+
+This RFC specifies the SDK work needed to support this feature.
+All SDKs are required to implement this feature.
```

Comment:
Interface Stability? (Volatile?)

Reply:
I'm unsure we typically specify stability level in RFCs? They seem better suited to being in CBDs. If we put it in the RFC then it means we later need to come back and update to remove it, which probably won't happen...